views/home.html

```diff
@@ -9,17 +9,17 @@
 			type="text/css"
 			href="https://Replit-2020.whippingdot.repl.co/static/style.css"
 		/>
-    <link
-    rel="shortcut icon"
-    href="favicon.ico"
-    type="image/x-icon"
-    />  
+		<link
+			rel="shortcut icon"
+			href="https://Replit-2020.whippingdot.repl.co/static/favicon_but_better.ico"
```

Comment:
use relative `./` not absolute

Reply:
Maybe, but it doesn't work. Did you forget the hours we used to try fixing this. It was our first problem

Reply:
It should be `rel="icon"` @whippingdot

Reply:
Try it yourself on the repl. Also @BD103 @JBYT27 @zplusfour and @darkdarcool joined the org. but @HuongTuon(Wumi4) left.